README.md

```diff
@@ -124,7 +124,7 @@ Pull requests on interesting tools/projects/resources are welcome.
 
 * [pre-commit](http://pre-commit.com/) - a framework for managing and maintaining multi-language pre-commit hooks from Yelp. Extensive support for multiple programming language.
 * [Overcommit](https://github.com/brigade/overcommit/) - a extendable Git hook manager written with Ruby.
-* [git-hooks](https://github.com/icefox/git-hooks/) - tool to manage project, user, and global Git hooks
```

Comment:
there is an alternative for icefox git-hooks located at https://github.com/git-hooks/git-hooks/